tests/cpp/builder_tests.cpp

```diff
@@ -141,6 +141,37 @@ RuleWithMetasWorks) {
 )", yaraFile->getTextFormatted());
 }
 
+TEST_F(BuilderTests,
+RuleWithEmptyStringMetasWorks) {
```

Comment:
Consider naming the test `RuleWithEmptyStringMetaValueWorks` as we actually check that an empty _value_ is permitted (an empty key is still forbidden).

Reply:
Thank you for your review, I have fixed the code as you suggested. The first of your tips will come as a separate PR.